configs/olimex-stm32-e407/uros_6lowpan/defconfig

```diff
@@ -0,0 +1,125 @@
+#
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
+#
+# You can use "make menuconfig" to make any modifications to the installed .config file.
+# You can then do "make savedefconfig" to generate a new defconfig file that includes your
+# modifications.
+#
+# CONFIG_ARCH_FPU is not set
```

Comment:
for any serious usage of the board, the FPU is necessary. therefore, I always try enable it, to provide a better basis for people's own developments.

Reply:
I've considered this defconfig as a probe of the concept that micro-ROS can communicate over 6LoWPAN radio. I mean something very basic, but there is no problem to add.